Plugins/Bitbucket/Settings.cs

```diff
@@ -26,7 +26,7 @@ public static Settings Parse(IGitModule gitModule, ISettingsSource settings, Bit
             var module = ((GitModule)gitModule);
 
             var remotes = module.GetRemotes()
-                .Select(r => module.GetSetting(string.Format(SettingKeyString.RemoteUrl, r)))
+                .Where(s => !string.IsNullOrWhiteSpace(s)).Distinct().Select(r => module.GetSetting(string.Format(SettingKeyString.RemoteUrl, r)))
```

Comment:
please have each command on a separate line, e.g.:

```
   .Where(...)
   .Distinct()
  .Select(...)
```

It makes it more readable